File: client-mobile-app/lib/models/online_object_info.dart
import 'package:json_annotation/json_annotation.dart';

part 'online_object_info.g.dart';

@JsonSerializable()
class OnlineObjectInfo {
  String objectId;
  String currentTrack;
  DateTime date;
  num secondsFromStart;

  OnlineObjectInfo({
    this.objectId,
    this.currentTrack,
    this.date,
    this.secondsFromStart,
  });

  factory OnlineObjectInfo.fromJson(Map<String, dynamic> json) =>
      _$OnlineObjectInfoFromJson(json);

  Map<String, dynamic> toJson() => _$OnlineObjectInfoToJson(this);
}

File: Backend/Player.ClientIntegration/Object/OnlineObjectInfo.cs
using System;

namespace Player.ClientIntegration.Object
{
    public class OnlineObjectInfo
    {
        public Guid ObjectId { get; set; }
        public string CurrentTrack { get; set; }
        public DateTime Date { get; set; }
        public int SecondsFromStart { get; set; }
    }
}